src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -44,7 +44,7 @@
 
 vsts_account_name = CliArgumentType(
     options_list=('--vsts-account-name',),
-    help='VSTS account name to create the build and release definitions. A new VSTS account is created if omitted or does not exist.'
+    help='VSTS account name to create the build and release definitions. If you have an existing VSTS account at myvstsaccount.visualstudio.com, then set this parameter value to myvstsaccount. A new VSTS account is created if parameter value is omitted or does not exist.'
 )
 
 vsts_project_name = CliArgumentType(
```

Comment:
GitHub's review won't let me put this comment where I *want* it (line 73), but...

Rather than have the same set of aliases for 'container release create' and 'container build create', since they share the same parameter names and definitions, you can just have a single set, and use `'container'` for the first parameter instead of `'container very specific scope'`.